File: BehavioralDesignPattern/ChainOfResponsibilityPattern/java/Driver.java
package BehavioralDesignPattern.ChainOfResponsibilityPattern.java;

import BehavioralDesignPattern.ChainOfResponsibilityPattern.java.AbstractHandler.Logger;

public class Driver {
    public static void main(String[] args){
        Logger logger = LoggerChain.getLoggerChain();

        logger.log(Logger.INFO, "This should only print in info");
        logger.log(Logger.DEBUG, "This should only print in Debug");
        logger.log(Logger.ERROR, "This should only print in Error");
    }
}

File: BehavioralDesignPattern/ChainOfResponsibilityPattern/python/Driver.py
from AbstractHandler.Logger import Logger
from LoggerChain import LoggerChain


if __name__ == "__main__":
    logger = LoggerChain.getLoggerChain()

    logger.log(Logger.INFO, "This is an INFO message")
    logger.log(Logger.DEBUG, "This is an DEBUG message")
    logger.log(Logger.ERROR, "This is an ERROR message")